src/components/checkbox/checkbox.test.js

```diff
@@ -0,0 +1,549 @@
+          "have.css",
+          "border-bottom-color",
+          "rgb(239, 103, 0)"
+        );
+      });
+
+      it("should render CheckboxGroup component with info ", () => {
+        CypressMountWithProviders(<CheckboxGroupComponent info />);
+
+        checkboxSvg().should(
+          "have.css",
+          "border-bottom-color",
+          "rgb(0, 96, 167)"
+        );
+      });
+
+      it("should render CheckboxGroup component with error message ", () => {
+        CypressMountWithProviders(
+          <CheckboxGroupComponent error="Error has occurred" />
+        );
+
+        checkboxGroupIcon()
+          .should("have.attr", "aria-label", "Error has occurred")
+          .should("have.attr", "data-element", "error");
+      });
+
+      it("should render CheckboxGroup component with warning message", () => {
+        CypressMountWithProviders(
+          <CheckboxGroupComponent warning="Warning has occurred" />
+        );
+
+        checkboxGroupIcon()
+          .should("have.attr", "aria-label", "Warning has occurred")
+          .should("have.attr", "data-element", "warning");
+      });
+
+      it("should render CheckboxGroup component with info message", () => {
+        CypressMountWithProviders(
+          <CheckboxGroupComponent info="Info has occurred" />
+        );
+
+        checkboxGroupIcon()
+          .should("have.attr", "aria-label", "Info has occurred")
+          .should("have.attr", "data-element", "info");
+      });
+
+      it("should render CheckboxGroup component with groupName", () => {
+        CypressMountWithProviders(<CheckboxGroupComponent />);
+
+        checkboxRole()
+          .eq(0)
+          .should("have.attr", "name", "checkbox_checkboxGroupName-Required");
+      });
+
+      it.each([
+        ["left", "flex-start"],
+        ["right", "flex-end"],
+      ])(
+        "should render CheckboxGroup component with inline legend aligned to %s",
+        (position, assertion) => {
+          CypressMountWithProviders(
+            <CheckboxGroupComponent
+              legend="CheckBox Legend"
+              legendWidth={20}
+              legendAlign={position}
+              legendInline
+            />
+          );
+          checkboxgroupLegend().should(
+            "have.css",
+            "justify-content",
+            assertion
+          );
+        }
+      );
+
+      it.each([20, 40])(
+        "should render CheckboxGroup component with inline legend width set to %s ",
+        (width) => {
+          CypressMountWithProviders(
+            <CheckboxGroupComponent
+              legend="CheckBox Legend"
+              legendWidth={width}
+              legendInline
+            />
+          );
+          checkboxgroupLegend().should("have.attr", "width", width);
+        }
+      );
+
+      it("should render CheckboxGroup component with children ", () => {
+        CypressMountWithProviders(
+          <CheckboxGroupComponent>Child 1 Child 2</CheckboxGroupComponent>
```

Comment:
those 2 values are accepted as 1 `string`, you already passed `<Checkbox>` into `<CheckboxGroup>`. Or you can pass some other element (as per documentation `node`)

Reply:
updated to pass a checkbox as a child